man/get_api_key.Rd

```diff
@@ -52,7 +52,7 @@ Delphi Epidata API Registration Form.
 \url{https://api.delphi.cmu.edu/epidata/admin/registration_form}
 }
 \seealso{
-\code{\link[usethis:edit_r_environ]{usethis::edit_r_environ()}} to automatically edit the \code{.Renviron}
-file; \code{\link[usethis:edit_r_profile]{usethis::edit_r_profile()}} to automatically edit the \code{.Rprofile}
+\code{\link[usethis:edit]{usethis::edit_r_environ()}} to automatically edit the \code{.Renviron}
+file; \code{\link[usethis:edit]{usethis::edit_r_profile()}} to automatically edit the \code{.Rprofile}
```

Comment:
How did this get updated without the associated .R file getting updated?

Reply:
Could you also add a blurb about `usethis::edit_r_environ()` into the warning message when no API key is set?